oci/archive/oci_transport.go

```diff
@@ -159,8 +159,11 @@ func (t *tempDirOCIRef) deleteTempDir() error {
 }
 
 // createOCIRef creates the oci reference of the image
-func createOCIRef(image string) (tempDirOCIRef, error) {
-	dir, err := ioutil.TempDir(tmpdir.TemporaryDirectoryForBigFiles(), "oci")
+func createOCIRef(image string, sys *types.SystemContext) (tempDirOCIRef, error) {
```

Comment:
By convention, the parameters go `context.Context, *types.SystemContext`, then others.